### ISSUE TITLE
Xaml navigation without animation isn

### ISSUE THREAD

**Issue by bares43** [PrismLibrary/Prism#1683]
### Steps to Reproduce

1. 
2. 
3. 

### Basic Information

- Version with issue:
- Last known good version:
- Xamarin.Forms version:
- IDE:

**Comment by brianlagunas** [comment]
Yeah, that was my fault because I used an old copy of the initial spec code snippets for the blog post. The final version of the feature did not have those properties in it.  This will be considered an enhancement

**Comment by brianlagunas** [comment]
For now, just create your own extension, add your desired properties, and then override `HandleNavigation` to modify the navigation call.

**Comment by bares43** [comment]
Thanks for answer. I've already created my own extension but it would be better if this could handle Prism itself. :)

**Comment by brianlagunas** [comment]
We accept PRs

**Comment by bares43** [comment]
I guess I should try it

**Comment by bares43** [comment]
So PR #1684 is created :)
